index.html

```diff
@@ -274,24 +274,26 @@ <h3>Success Criteria</h3>
 In order to advance to
 <a href="https://www.w3.org/Consortium/Process//#RecsPR"
 title="Proposed Recommendation">Proposed Recommendation</a>, each specification
-is expected to have
+will fulfill the
+<a href="http://www.w3.org/Consortium/Process/#implementation-experience">implementation experience required by the W3C Process</a>
+as follows:
           </p>
 
           <ul>
             <li>
-<a href="https://www.w3.org/Consortium/Process/#implementation-experience">At least two independent implementations</a>
-of each of feature defined in the specification, and
+The Working Group will seek evidence of independent interoperable uses of the
+data model from at least two independent implementations of each of feature
```

Comment:
`each of feature`
-> `each feature`

Reply:
Thanks!
fixed in [656ab1b](https://github.com/w3c-ccg/did-wg-charter/pull/39/commits/656ab1b47de069214d54d2912a72539385b41b44)